src/app/shared/models/lightning.ts

```diff
@@ -0,0 +1,5 @@
+export interface LightningDetails {
```

Comment:
For future proofing, we could include BOLT11 in this interface name.

Your choice on design.

Reply:
Sublass later if needed works as well ⚡